### PR TITLE
Adding doc files

### PR DESCRIPTION
**Pull request by vahid-ahmadi**
I create a branch to add documentation files.

### CODE REVIEW

**Comment by vahid-ahmadi** [comment]
Fixes #20

File: docs/methodology.ipynb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    "\n",
+    "## Data Source Selection and Acquisition\n",
+    "\n",
+    "The foundation of our calibration process rests on carefully selected authoritative data sources. Nomis serves as our primary source for earnings data, providing comprehensive information on income deciles split by sex and employment status (full-time/part-time). This granular breakdown enables detailed analysis of earning patterns across different demographic groups. We supplement this with HMRC income data, which serves as our baseline for EFRS calibration. The Annual Survey of Hours and Earnings (ASHE) provides additional reliable income targets, drawing strength from its basis in administrative data from HMRC PAYE firms. This combination of sources ensures a robust foundation for our subsequent analysis and calibration steps.\n",
+    "\n",
+    "## Earnings Distribution Integration\n",
```
Comment (nikhilwoodruff) [inline docs/methodology.ipynb:15]: GPT nit: every-word-capitalised headers

File: docs/methodology.ipynb [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    "\n",
+    "The integration of earnings distributions forms a crucial second step in our methodology. This process involves synthesizing the various data sources into a coherent framework that can be used for calibration. We begin by aligning local area statistics with national benchmarks, ensuring consistency across different geographic levels. The integration process carefully considers the relationships between different income bands and their distributions across regions. This step is particularly important as it sets the stage for creating accurate calibration targets that reflect both local variations and national patterns in income distribution.\n",
+    "\n",
+    "## Missing Data Imputation\n",
```
Comment (nikhilwoodruff) [inline docs/methodology.ipynb:19]: This section is unclear to me- could you be more specific about the things we are doing to which datasets?

File: docs/methodology.ipynb [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    "\n",
+    "## Constituency Mapping Framework\n",
+    "\n",
+    "The fourth step involves creating a comprehensive mapping framework to address the evolution of constituency boundaries. We have developed detailed mapping matrices that establish clear relationships between 2010 and 2024 constituencies. This framework ensures continuity in our analysis despite geographic boundary changes, allowing us to track and compare data across different time periods. The mapping process carefully considers the implications of boundary changes on income distribution patterns and adjusts the calibration accordingly.\n",
+    "\n",
+    "## Income Percentile Calibration\n",
```
Comment (nikhilwoodruff) [inline docs/methodology.ipynb:27]: This is not actually last right, isn't it pretty early on?

File: docs/methodology.ipynb [hunk 4]
```diff
@@ -0,0 +1,58 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Methodology\n",
+    "\n",
+    "This chapter outlines the comprehensive methodology used to calibrate income distributions across constituencies. Our approach combines multiple data sources and statistical techniques to ensure accurate representation of earnings across different geographical areas and demographic segments. The process consists of six major steps, each building upon the previous to create a robust calibration framework.\n",
```
Comment (nikhilwoodruff) [inline docs/methodology.ipynb]: Claude has a natural bias for being a bit self-complimentary. Could you tone it down a bit?

**Comment by MaxGhenis** [comment]
Need to mention the issue in the original comment to link it

**Pull request merged**